File: fizzbuzz.clj
(defn evl 
  [string default]
  (if (clojure.string/blank? string) default string))

(defn fizzbuzz
  [i]
  (evl (str (if (= (mod i 3) 0) "fizz" "") (if (= (mod i 5) 0) "buzz" "")) i))

(doseq [item (range 1 16)] (println (fizzbuzz item)))

File: fizzbuzz.scala
object FizzBuzz {
  def fizzbuzz(i: Int): String = {
    (i % 3 match { case 0 => "fizz" case x => "" }).concat(i % 5 match { case 0 => "buzz" case x => "" }) match { case "" => i.toString() case x => x }
  }
  def main(args: Array[String]): Unit = {
    List.range(1, 15 + 1).map(fizzbuzz).foreach(println)
  }
}
